pkg/autoscaler/config.go

```diff
@@ -48,10 +48,7 @@ type Config struct {
 	PanicWindow    time.Duration
 	TickInterval   time.Duration
 
-	ScaleToZeroThreshold   time.Duration
 	ScaleToZeroGracePeriod time.Duration
```

Comment:
I don't see you using this in the autoscaler.  How do we ensure that 1) we wait some time after the last request before updating the Istio routes and 2) we wait longer before scaling the deployment to zero?

Reply:
The "wait some time after the last request" is implicit after this change. It will only happen after the window contains straight 0s. This brings me to another problem though as we might have flappy route-state when scaling from 0 due to this.

I'll rework accordingly

/hold